conf/messages.cy

```diff
@@ -120,17 +119,14 @@ AddBenefits.CY.Caution.p12.3 = (sy’n dechrau ar 6 Ebrill {0})
 AddBenefits.noselection.error = Mae’n rhaid i chi ddewis o leiaf 1 buddiant neu draul cyn i chi allu mynd yn eich blaen
 
 AddBenefits.Confirm.Single.Title = Cofrestru buddiant neu draul
```

Comment:
I don't see use of the key `AddBenefits.Confirm.Single.Title` anywhere in the code so remove from here

Reply:
Done